### PR TITLE
Make comment up/down vote consistent after you vote

### PR DESCRIPTION
**Pull request by vijaykramesh**
this fixes https://github.com/dessalines/jerboa/issues/662

it makes it so for a comment you don't see the upvote/downvote specific count unless you vote - and then if you upvote or downvote you'll see both specific counts.

currently as #662 describes you only see this when you downvote, if you upvote you don't see it.

### CODE REVIEW

File: app/src/main/java/com/jerboa/ui/components/comment/CommentNode.kt [hunk 1]
```diff
@@ -565,7 +565,7 @@ fun CommentFooterLine(
                 item = commentView,
                 type = VoteType.Upvote,
                 onVoteClick = onUpvoteClick,
-                showNumber = (instantScores.downvotes != 0),
+                showNumber = (instantScores.myVote?.let { it != 0 } ?: let { false }),
```
Comment (dessalines) [inline app/src/main/java/com/jerboa/ui/components/comment/CommentNode.kt:568]: Might have been a bug someone slipped in, but really it should always show the upvotes, no matter what. So this should just be `showNumber = true`

The downvotes below, should only be shown, if there are downvotes, so `instantScores.downvotes != 0`

It doesn't have anything to do with whether *you* voted or not.

Comment (vijaykramesh) [reply app/src/main/java/com/jerboa/ui/components/comment/CommentNode.kt:568]: That is how the number on the right works but this is for the numbers next to the arrows for the comments, currently it is only showing up if you downvote, if you upvote they don't show. But neither show until you vote 

You are saying always show the upvote number (even before voting) but then show downvote if they click down?

Comment (vijaykramesh) [reply app/src/main/java/com/jerboa/ui/components/comment/CommentNode.kt:568]: It looks like lemmy ui does the same. Always show the up-down number on the right, only show the numbers by the arrow if you downvote, and never show the individual numbers if you upvote. My proposal would be always show the specific up and down numbers once you vote (this is how it works on the top level post up/down buttons)

Comment (vijaykramesh) [reply app/src/main/java/com/jerboa/ui/components/comment/CommentNode.kt:568]: Assuming you agree I can take a stab at changing this in lemmy-ui to match too

Comment (dessalines) [reply app/src/main/java/com/jerboa/ui/components/comment/CommentNode.kt:568]: Ohhh wait this is for the footer line.

Okay this is hard to articulate, but if there are no downvotes, it should only show the score, on the top right. It would be pointless to show `+5` on the top right, and `+5` on the footer bar.

If there are downvotes, this changes.

The score would then be `+4` on the top right, but the comment footer should now show `+5, -1` . I'm pretty sure this is how it already works in both lemmy-ui and jerboa.

Comment (vijaykramesh) [reply app/src/main/java/com/jerboa/ui/components/comment/CommentNode.kt:568]: Ahhh ok I follow now. I don't think it is actually working like that but let me test and see

Comment (vijaykramesh) [reply app/src/main/java/com/jerboa/ui/components/comment/CommentNode.kt:568]: I finally had time to see exactly what you mean and can confirm correct behavior in lemmy ui. (Sorry posting from my phone I'll go on my computer later to resize this)

Tldr if there are prexisting downvotes you see the detailed up/down count and total on the right. For that case if you up/down you keep seeing all 3 numbers 

If there are no down you only ever see right total. And if you upvote in this case still no up/down breakout needed or shown.

I'll double check jerboa later and I think will close this and the issue that came out of a !jerboa@lemmy.ml thread. Thanks for helping validate
